powerauth-java-crypto/src/main/java/io/getlime/security/powerauth/crypto/lib/util/ECPublicKeyFingerprint.java

```diff
@@ -90,7 +90,7 @@ public static String compute(ECPublicKey devicePublicKey, ECPublicKey serverPubl
             }
             int index = hash.length - 4;
             int number = (ByteBuffer.wrap(hash).getInt(index) & 0x7FFFFFFF) % (int) (Math.pow(10, PowerAuthConfiguration.FINGERPRINT_LENGTH));
-            return String.format("%0" + PowerAuthConfiguration.SIGNATURE_LENGTH + "d", number);
+            return String.format("%0" + PowerAuthConfiguration.FINGERPRINT_LENGTH + "d", number);
```

Comment:
This change fixes some previous copy-paste bug. Both constants have equal value, but with different meaning.